app/javascript/components/game_map.js

```diff
@@ -0,0 +1,63 @@
+import { Axis } from './axis'
```

Comment:
Can the default export be used for this style of class instead of a named export?

Reply:
Not now